packrat/src/main/java/com/github/reflectoring/infiniboard/packrat/source/SourceData.java

```diff
@@ -15,6 +15,10 @@
 
     private Map<String, Object> data;
 
+    public SourceData() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1186)